CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Changelog
 
 - Bump CUDA base image version to 12.3.2
+- Set MPS memory limit by device index and not device UUID. This is a workaround for an issue where
+  these limits are not applied for devices if set by UUID.
```

Comment:
I know we have MIG support for MPS disabled at the moment, but what implications is this going to have once we want to enable it later?

Reply:
Since we have to start an MPS control daemon per MIG device and we would explicitly set `CUDA_VISIBLE_DEVICES` in this case, we would always be referring to `index == 0` here. (or `index == 0:0` assuming that the daemon supports "nested" indices).

Reply:
It assume it does not support nested indices. That is something I came up with for NVIDIA_VISIBLE_DEVICES, and I don't think anything else adopted it.

Reply:
If the nested indices is not a valid convention, does this mean we need to deploy one container per MIG device -- where each container launches an MPS control daemon with `CUDA_VISIBLE_DEVICES=0`?

Reply:
Unfortunately, yes. See https://docs.nvidia.com/datacenter/tesla/mig-user-guide/index.html#cuda-mps